pos_barcode_tare/views/pos_config_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+
+<odoo noupdate="1">
```

Comment:
- the view view_pos_config_form should be in a classical ``odoo`` tag. (without no update) because view should allways updated.

- the barcode rule should be in a file like /data/barcode_rule.xml in  ``odoo noupdate=1`` because this data should be customizable by the user. (changing pattern, etc...)

Reply:
Thank you for the explanation, I will do so.

Reply:
Done!